src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+$bodyFont: 'Roboto';
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), using the universal selector (*) is highly discouraged due to its potential performance issues and unintended style resets. It's better to apply zero margin and padding to specific elements, or use a more targeted reset approach.